python/ql/test/library-tests/frameworks/twisted/response_test.py

```diff
@@ -0,0 +1,75 @@
+from twisted.web.server import Site, Request, NOT_DONE_YET
+from twisted.web.resource import Resource
+from twisted.internet import reactor, endpoints, defer
+
+
+root = Resource()
+
+class Now(Resource):
+    def render(self, request: Request): # $ requestHandler
+        return b"now" # $ HttpResponse mimetype=text/html responseBody=b"now"
+
+
+class AlsoNow(Resource):
+    def render(self, request: Request): # $ requestHandler
+        request.write(b"also now") # $ HttpResponse mimetype=text/html responseBody=b"also now"
+        return b"" # $ HttpResponse mimetype=text/html responseBody=b""
+
+
+def process_later(request: Request):
+    print("process_later called")
+    request.write(b"later") # $ MISSING: responseBody=b"later"
```

Comment:
Why is this missing?

Reply:
Because `process_later` is invoked through `reactor.callLater(1, process_later, request)`, and we currently have no way to simulate this call happening.

Reply:
Ah, I see now. `TwistedResourceRequestHendler` would have to close under such delegating calls. I think it is fine to just have the test case documenting the limitation for now.